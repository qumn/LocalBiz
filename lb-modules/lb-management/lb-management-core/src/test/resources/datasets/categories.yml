lb_category:
  - cat_id: 1
    mid: 1
    name: "cat name1"
    priority: 1
  - cat_id: 2
    mid: 1
    name: "cat name2"
    priority: 2
  - cat_id: 3
    mid: 2
    name: "cat name3"
    priority: 1
