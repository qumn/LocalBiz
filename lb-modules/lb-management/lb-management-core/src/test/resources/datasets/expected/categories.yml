lb_category:
  - cat_id: 2
    mid: 1
    name: "cat name2"
    priority: 2
  - cat_id: 3
    mid: 2
    name: "cat name3"
    priority: 1